src/matrices.jl

```diff
@@ -1,3 +1,5 @@
+using Statistics
+using ChaosTools:linear_region
```

Comment:
You can't add `ChaosTools` as a dependency of RecurrenceAnalysis. You can just copy the `linear_region` code, but I think it is probably better to move `linear_region` to DelayEmbeddings.jl, as it is a more basic package and the method is generally useful. 

I'll do this ASAP.